appserver/ejb/ejb-http-remoting/client/src/main/java/fish/payara/ejb/http/client/RemoteEJBContextFactory.java

```diff
@@ -79,7 +79,8 @@
     public static final String JAXRS_CLIENT_HOSTNAME_VERIFIER = "fish.payara.hostnameVerifier";
     public static final String JAXRS_CLIENT_EXECUTOR_SERVICE = "fish.payara.executorService";
     public static final String JAXRS_CLIENT_CONNECT_TIMEOUT = "fish.payara.connectTimeout";
-
+    public static final String JAXRS_CLIENT_SERIALIZATION = "fish.payara.serialization";
+    public static final String JAXRS_CLIENT_VERSION = "fish.payara.version";
```

Comment:
I think both of these probably need to be a bit more specific.
fish.payara.version particularly sounds more like a property to get the version of Payara itself rather than the version of the JAX-RS client.

Reply:
I agree on that one, also JAXRS_CLIENT_VERSION seems to be telling different story. Probably it should  mention protocol version

Reply:
I thought the same myself but the existing ones didn't use some namespacing so I did not add one. In hindsight it might have been nicer to have a common namespace like `fish.payara.ejb.http` for all of them. Any particular suggestions on the two names?

Reply:
Actually I completely overlooked those other properties - yeah it probably would be better to have a proper namespace for them.

TBH I like the namespace you gave, `fish.payara.ejb.http`, but I recognise that it's perhaps a bit verbose. I think the names are fine if you use that namespace though (e.g. `fish.payara.ejb.http.version`).

I kind of agree with Patrik in that the variable name for _JAXRS_CLIENT_VERSION_ is a bit misleading as it seems to be more to do with the EJB HTTP protocol version (unless I'm misreading it).

Reply:
Changed the name and properties.